readabilitySAX.js

```diff
@@ -206,7 +206,10 @@ Readability.prototype._settings = {
 	linksToSkip: {},	//pages that are already parsed
 	//pageURL: null,	//URL of the page which is parsed
 	//type: "html",		//default type of output
-	resolvePaths: false
+	resolvePaths: false,
+    includeEmbeds: function(src) {
```

Comment:
This shouldn't replace the `embed` handling, but might be added as a separate option.